.github/workflows/ensure-jdk8.yml

```diff
@@ -1,60 +0,0 @@
-# Licensed to the Apache Software Foundation (ASF) under one
-# or more contributor license agreements.  See the NOTICE file
-# distributed with this work for additional information
-# regarding copyright ownership.  The ASF licenses this file
-# to you under the Apache License, Version 2.0 (the
-# "License"); you may not use this file except in compliance
-# with the License.  You may obtain a copy of the License at
-#
-#   http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing,
-# software distributed under the License is distributed on an
-# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-# KIND, either express or implied.  See the License for the
-# specific language governing permissions and limitations
-# under the License.
-
-name: Ensure JDK-8
-
-on:
-  push:
-  pull_request:
-
-jobs:
-  jdk11-jdk8:
-    name: "Compile with JDK-11 and test something on JDK-8"
-    runs-on: ubuntu-18.04
-    env:
-      OPTS: -Dcluster.config=basic -Dtest-unit-sys-prop.ignore.random.failures=true
-    steps:
-      - uses: actions/checkout@v2
-
-      - name: Caching dependencies
-        uses: actions/cache@v2
-        with:
-          path: ~/.hgexternalcache
-          key: ${{ runner.os }}-${{ hashFiles('*/external/binaries-list', '*/*/external/binaries-list') }}
-          restore-keys: ${{ runner.os }}-
-
-      - name: Set up JDK 8
-        uses: actions/setup-java@v2
-        with:
-            distribution: 'zulu'
-            java-version: 8
-
-      - name: Record JDK8
-        run: echo "$JAVA_HOME" | tee `pwd`/jdk8
-
-      - name: Set up JDK 11
-        uses: actions/setup-java@v2
-        with:
-            distribution: 'zulu'
-            java-version: 11
-
-
-      - name: Build
-        run: ant $OPTS build
-
-      - name: Test java.source.base on JDK-8
```

Comment:
I am not sure what replaces testing on JDK8? I am against deleting any jobs that perform testing on JDK8 unless they are replaced by appropriate alternative.

Reply:
Testing on JDK 8 is covered by travis: https://github.com/apache/netbeans/blob/0665eb12aaa64cbe3b7bae657f46594a579a41cc/.travis.yml#L383